src/main/java/org/broadinstitute/hellbender/engine/FeatureDataSource.java

```diff
@@ -349,7 +356,7 @@ public FeatureDataSource(final FeatureInput<T> featureInput, final int queryLook
         }
     }
 
-    protected static FeatureReader<VariantContext> getGenomicsDBFeatureReader(final String path, final File reference) {
+    protected static FeatureReader<VariantContext> getGenomicsDBFeatureReader(final String path, final File reference, final boolean callGenotypes) {
```

Comment:
You've threaded this boolean all the way down here, and it doesn't appear to actually be used? Am I missing something?

Reply:
Also, can I suggest that we introduce some sort of `GenomicsDBSettings` object, and pass that down through the engine classes instead of this boolean, just in case we need to thread through additional GenomicsDB settings in the future?

Reply:
Ugh, maybe I botched the rebase.  I'll take a look.  And I can certainly make a settings object.

Reply:
I made a GenomicsDBOptions (or I can call it Settings if you really want) class, which the VariantWalker constructs, but is taken care of under the hood for other walkers.